src/main/scala/org/broadinstitute/dsde/rawls/model/WorkspaceModel.scala

```diff
@@ -23,31 +24,22 @@ case class WorkspaceName(
   def path : String = "workspaces/" + namespace + "/" + name
 }
 
-@ApiModel(value = "WorkspaceShort")
-case class WorkspaceShort(
-                      @(ApiModelProperty@field)(required = true, value = "The namespace the workspace belongs to")
-                      namespace: String,
-                      @(ApiModelProperty@field)(required = true, value = "The name of the workspace")
-                      name: String,
-                      @(ApiModelProperty@field)(required = true, value = "The date the workspace was created in yyyy-MM-dd'T'HH:mm:ssZZ format")
-                      createdDate: DateTime,
-                      @(ApiModelProperty@field)(required = true, value = "The user who created the workspace")
-                      createdBy: String) extends Identifiable {
-  def path : String = "workspaces/" + namespace + "/" + name
-}
-
 @ApiModel(value = "Workspace")
 case class Workspace (
                       @(ApiModelProperty@field)(required = true, value = "The namespace the workspace belongs to")
+                      @(VertexProperty@field)
                       namespace: String,
                       @(ApiModelProperty@field)(required = true, value = "The name of the workspace")
+                      @(VertexProperty@field)
                       name: String,
                       @(ApiModelProperty@field)(required = true, value = "The date the workspace was created in yyyy-MM-dd'T'HH:mm:ssZZ format")
+                      @(VertexProperty@field)
                       createdDate: DateTime,
                       @(ApiModelProperty@field)(required = true, value = "The user who created the workspace")
+                      @(VertexProperty@field)
                       createdBy: String,
                       @(ApiModelProperty@field)(required = true, value = "Entities in the workspace, first key: entity type, second key: entity name")
-                      entities: Map[String, Map[String, Entity]]
+                      attributes: Map[String, Attribute]
```

Comment:
If it's attribute, can't a workspace now have an AttributeList or AttributeReference...?